### PR TITLE
Blueprint azure

### PR DESCRIPTION
**Pull request by christiancbo**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ New Features
     • Ubuntu 18.04
     • SUSE Enterprise Linux Server 12
     • openSUSE 42
+* Blueprint deployment has been improved. Users can now deploy blueprint appliance on Microsoft Azure Plublic Cloud.
```
Comment (emuus) [inline changelog.rst]: appliances (plural)

Comment (AliceAlbano) [reply changelog.rst]: Maybe we can remove appliances: Users can now deploy blueprints on Microsoft Azure Plublic Cloud.
(not sure about that, just an idea)

Comment (christiancbo) [reply changelog.rst]: I put "blueprint appliance" to have the notion mono and multi nodes, but I can change if you think itś better with out "appliance"

Comment (AliceAlbano) [reply changelog.rst]: I think it is better without "appliances", it could be confusing for the user to have "appliances" and  "blueprint appliances"

**Pull request merged**